decoding.py

```diff
@@ -258,8 +274,32 @@ def cross_validation_across_time(data_x, data_y, clf, add_null_data=False,
         all_probas[idxs_test] = probas
 
         preds = np.argmax(probas, -1)
-
-        accuracy = (preds == test_y[:, None]).mean(axis=0)
+        
+        if metric == "accuracy": 
+            accuracy = (preds == test_y[:, None]).mean(axis=0)
+        else:
```

Comment:
you can also do this using the `top_k_accuracy_score(k=1)` :) makes it a bit cleaner